spec/lib/faraday/zipkin-tracer_spec.rb

```diff
@@ -17,29 +17,29 @@ def on_complete
   end
 
   # Ruby 1.8 didn't support \h
-  HEX_REGEX = RUBY_VERSION >= "1.9.2" ? '\h' : '[0-9a-fA-F]'
+  HEX_REGEX = RUBY_VERSION >= '1.9.2' ? '\h' : '[0-9a-fA-F]'
```

Comment:
Hmm, yea it's a bit strange. Thanks for the idea though - I'll keep the naming as is, but switch to the 1.9 syntax